src/index.html

```diff
@@ -7,11 +7,150 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>HTML Forms</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="block_input">
+        <legend>Personal information</legend>
+        <label class="line__input">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+          >
+        </label>
+        <label class="line__input">
+          Name:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="line__input">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="line__input">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthdate"
+          >
+        </label>
+        <label class="line__input__last">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block_input">
+        <legend>Registration</legend>
+        <label class="line__input">
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="line__input__last">
+          Password:
+          <input
+            type="password"
+            name="pass"
+            minlength="5"
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block_input">
+        <legend>An interesting fact about you!</legend>
+        <label class="line__input">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            value="Yes"
+          >
+          Yes
+          <input
+            type="radio"
+           name="cats"
+            value="No"
+          >
+          No
+        </label>
+        <label class="line__input">
+          What is your favorite colour?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="line__input">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+          >
+        </label>
+        <label class="line__input">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            required
+            multiple>
```

Comment:
```suggestion
          <select
            name="cars"
            required
            multiple
          >
```
Closing tag should be under the opening
![image](https://user-images.githubusercontent.com/96409563/223971985-5285fad8-a8fe-4331-a98e-823fbb3089de.png)